bin/serverless.js

```diff
@@ -7,7 +7,7 @@ EvalError.$serverlessCommandStartTime = process.hrtime();
 
 const nodeVersion = Number(process.version.split('.')[0].slice(1));
 
-if (nodeVersion < 10) {
+if (nodeVersion < 12) {
```

Comment:
@vaidyamilind  we should not do that, as that removes support for v10, while in context of this we should just deprecate it.

Please see link in an issue description that suggests how to configure the deprecation

Reply:
Currently, for all < v10, we are showing Initialization error and exiting.
Do we need an OUTDATED_NODEJS log for version within range v10 and v12? or for all version which is < 12

Reply:
Please see an issue description. I've linked there specifically an implementation we expect.

In v2 we just need to show deprecation log, and let v10 deployments work without issues

Reply:
@medikoo Updated